core/integer.rbs

```diff
@@ -1263,43 +1214,7 @@ class Integer < Numeric
   #
   # Raises an exception if `base` is out of range.
   #
-  def to_s: () -> String
-          | (2) -> String
-          | (3) -> String
-          | (4) -> String
-          | (5) -> String
-          | (6) -> String
-          | (7) -> String
-          | (8) -> String
-          | (9) -> String
-          | (10) -> String
-          | (11) -> String
-          | (12) -> String
-          | (13) -> String
-          | (14) -> String
-          | (15) -> String
-          | (16) -> String
-          | (17) -> String
-          | (18) -> String
-          | (19) -> String
-          | (20) -> String
-          | (21) -> String
-          | (22) -> String
-          | (23) -> String
-          | (24) -> String
-          | (25) -> String
-          | (26) -> String
-          | (27) -> String
-          | (28) -> String
-          | (29) -> String
-          | (30) -> String
-          | (31) -> String
-          | (32) -> String
-          | (33) -> String
-          | (34) -> String
-          | (35) -> String
-          | (36) -> String
```

Comment:
These were probably to represent that `integer.to_s(69)` is an error.
But this error is literally not a type error:
```ruby
42.to_s 'a' # TypeError
42.to_s 69  # ArgumentError
```
Not to mention how’d this error arise in production…